homeassistant/components/deconz/alarm_control_panel.py

```diff
@@ -124,20 +151,21 @@ def async_update_callback(self) -> None:
     @property
     def state(self) -> str | None:
         """Return the state of the control panel."""
-        return DECONZ_TO_ALARM_STATE.get(self._device.state)
+        assert self._device.panel
```

Comment:
Change of behaviour?

Reply:
Both of these lines could be removed it seems